pkg/rancher-desktop/pages/marketplace/details.vue

```diff
@@ -0,0 +1,364 @@
+              :key="imageStats.Created"
+              class="extension-content-sidebar-version-element-platform"
+            >
+              <p>
+                <b>OS:</b>
+                {{ imageStats.OS }} | {{ imageStats.Arch }}
+              </p>
+              <p>
+                <b>Updated:</b>
+                {{ formatDate(imageStats.Created) }}
+              </p>
+            </div>
+          </div>
+
+          <div class="extension-content-sidebar-version-element">
+            <p>{{ t('marketplace.sidebar.links') }}</p>
+            <ul>
+              <li v-for="links in extensionDetails.links" :key="links.title">
+                <a :href="links.url" target="_blank">
+                  {{ links.title }}
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { Banner } from '@rancher/components';
+import dayjs from 'dayjs';
+
+import LoadingIndicator from '@pkg/components/LoadingIndicator.vue';
+import demoMetadata from '@pkg/utils/_demo_metadata.js';
+
+export default {
+  name:       'marketplace-details',
+  components: { LoadingIndicator, Banner },
+  data() {
+    return {
+      extension:        this.$route.params.slug,
+      extensionDetails: null,
+      metadata:         null,
+      image:            this.$route.params.image,
+      loading:          false,
+      isInstalled:      null,
+      error:            null,
+      response:         null,
+      credentials:      null,
+    };
+  },
+
+  async fetch() {
+    this.credentials = await this.$store.dispatch(
+      'credentials/fetchCredentials',
+    );
+
+    fetch(`http://localhost:${ this.credentials?.port }/v1/extensions`, {
+      headers: new Headers({
+        Authorization: `Basic ${ window.btoa(
+          `${ this.credentials?.user }:${ this.credentials?.password }`,
+        ) }`,
+        'Content-Type': 'application/x-www-form-urlencoded',
+      }),
+    }).then((response) => {
+      if (!response.ok) {
+        return;
+      }
+
+      response.json().then((res) => {
+        this.isInstalled = Object.keys(res).includes(this.extension);
+      });
+    });
+  },
+
+  computed: {
+    htmlContent() {
+      if (!this.metadata) {
+        return;
+      }
+
+      return (
+        this.metadata?.LatestVersion.Labels[
+          'com.docker.extension.detailed-description'
+        ].replace('h1', 'h2') || ''
+      );
+    },
+
+    getPreviewImage() {
+      if (!this.metadata) {
+        return;
+      }
+
+      // TODO: Maybe we want to use a gallery here ?
+      const screenshots =
+        this.metadata?.LatestVersion.Labels['com.docker.extension.screenshots'];
+
+      return JSON.parse(screenshots)[0].url || '';
+    },
+  },
+
+  watch: {},
+
+  mounted() {
+    if (!this.extension) {
+      this.$router.push('/marketplace');
+    }
+
+    // TODO: get metadata from the backend
```

Comment:
I think we should have an Issue on our board that we can use to link to this TODO item